tests/storage/test_mount.py

```diff
@@ -0,0 +1,43 @@
+import threading
+import subprocess
+import time
+import sys
+
+sys.path.append("..")
+import _thread as thread
+import os
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from TestBase import Test
+import argparse
+
+
+class Mount(Test):
+    """Checks for eos mount points"""
+
+    def __init__(self):
+        self.ref_test_name = "mount"
+        super().__init__()
+
+    def check_mount(self):
+        olddir = os.getcwd()
+        os.chdir('/')
+        first_command = subprocess.Popen(["mount", "-l"], stdout=subprocess.PIPE)
+        second_command = subprocess.Popen(["grep", "eos"], stdin=first_command.stdout)
+        if second_command:
```

Comment:
This only checks that 'eos' is somewhere in the command of `mount -l | grep eos`, correct?

Reply:
Yes, I'm checking the specific mount points in the `test_mount_sanity.py` file